app/Actions/PushToGitHub.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Actions;
+
+use App\ConsoleWriter;
+use App\Shell;
+
+class PushToGitHub
+{
+    protected $shell;
+    protected $consoleWriter;
+
+    public function __construct(Shell $shell, ConsoleWriter $consoleWriter)
+    {
+        $this->shell = $shell;
+        $this->consoleWriter = $consoleWriter;
+    }
+
+    public function __invoke()
+    {
+        if (! (config('lambo.store.push_to_github'))) {
+            return;
+        }
+
+        $this->consoleWriter->logStep('Pushing new project to GitHub');
+
+        $branch = config('lambo.store.branch');
+        $process = $this->shell->execInProject("git -c credential.helper= -c credential.helper='!gh auth git-credential' push -u origin {$branch}");
```

Comment:
This is unfamiliar to me. Do we have to do this in order to sync Git's credential's and gh's credentials? And do we need the blank `-c credential-helper=` first?

Reply:
I've removed this. Git will ask for authentication if appropriate.